.github/workflows/phpunit-tests.yml

```diff
@@ -36,7 +36,7 @@ jobs:
   #
   test-with-mysql:
     name: PHP ${{ matrix.php }}
-    uses: WordPress/wordpress-develop/.github/workflows/phpunit-tests-run.yml@trunk
+    uses: youknowriad/wordpress-develop/.github/workflows/phpunit-tests-run.yml@update/packages-65
```

Comment:
This can also be used for testing (personally, I just find it more reliable).

```suggestion
    uses: ./.github/workflows/phpunit-tests-run.yml
```